Microsoft.Toolkit.Uwp.SampleApp/Pages/SampleController.xaml.cs

```diff
@@ -224,24 +224,26 @@ protected override async void OnNavigatedTo(NavigationEventArgs e)
                     _onlyDocumentation = true;
                 }
 
-                DataContext = CurrentSample;
-
-                var propertyDesc = CurrentSample.PropertyDescriptor;
-
                 InfoAreaPivot.Items.Clear();
 
-                if (propertyDesc != null)
+                if (CurrentSample.HasXAMLCode)
                 {
-                    _xamlRenderer.DataContext = propertyDesc.Expando;
-                }
+                    // Load Sample Properties before we load sample (if we haven't before)
+                    await CurrentSample.PreparePropertyDescriptorAsync();
 
-                if (propertyDesc != null && propertyDesc.Options.Count > 0)
-                {
-                    InfoAreaPivot.Items.Add(PropertiesPivotItem);
-                }
+                    // We only have properties on examples with live XAML
+                    var propertyDesc = CurrentSample.PropertyDescriptor;
+
+                    if (propertyDesc != null)
+                    {
+                        _xamlRenderer.DataContext = propertyDesc.Expando;
+                    }
+
+                    if (propertyDesc != null && propertyDesc.Options.Count > 0)
```

Comment:
nit: This can be simplified to `if (propertyDesc?.Options.Count > 0) {... }`

Reply:
```suggestion
                    if (propertyDesc?.Options.Count > 0)
```